modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -9,7 +9,7 @@
 Different channel and broker types have their own behavior patterns that are followed for event delivery.
 
 [id="serverless-event-delivery-component-behaviors-kafka_{context}"]
-== Knative Kafka channels and brokers
+== Apache Kafka channels and brokers
```

Comment:
If we say "Apache kafka channel" or "apache kafka broker", I think this is misleading.

apache kafka (own its own, without knative), defines a broker, so, if we say "apache kafka broker", I get the impression we mean that broker. But we do not mean it, that is why we say most of the time "Knative broker for Apache Kafka"

Similar with channels: Perhaps Knative channels for Apache Kafka? 

(mostly to avoid misunderstandings)

Reply:
@matzew I have made those changes as suggested